package-lock.json

```diff
@@ -0,0 +1,4948 @@
+{
```

Comment:
We use yarn not npm for this repo, so there should be no package-lock.json (only yarn.lock)